geode-redis/src/main/java/org/apache/geode/redis/internal/netty/ExecutionHandlerContext.java

```diff
@@ -326,7 +326,7 @@ private void executeCommand(Command command) {
         channelInactive(command.getChannelHandlerContext());
       }
     } catch (Exception e) {
-      logger.warn("Execution of Redis command {} failed: {}", command, e);
+      logger.warn("Execution Redis command {} failed: {}", command, e);
```

Comment:
Execution **of** Redis command {} failed: {} - I think the 'of' accidentally was deleted